content/blog/2021-06-25-june-21-community-gems.md

```diff
@@ -0,0 +1,132 @@
+---
+title: June '21 Community Gems
+date: 2021-06-30
+description: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: DVC pipeline configs, working with remotes,
+  file handling and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: DVC pipeline configs, working with remotes,
+  file handling and more.
+picture: 2021-06-30/gems-cover.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/june-21-community-gems/779
+tags:
+  - Community
+  - Plots
+  - Pipelines
+  - CML
+  - Git
+---
+
+### [Q: Is it possible to plot multiple experiments together?](https://discord.com/channels/485586884165107732/563406153334128681/834387923482181653)
+
+You can use experiment names in the `dvc plots` commands. You need to use the
+`diff` command to compare multiple plots. Try
+`dvc plots diff exp-6ef18 exp-b17b4 exp-26e88`.
+
+Thanks to @PythonF from Discord for asking this question that led to this Gem!
+💎
+
+### [Q: Where is the list of experiment being pushed in Git when I run `dvc exp push`?](https://discord.com/channels/485586884165107732/563406153334128681/837773937390649364)
+
+It uses custom Git refs internally, similar to the way GitHub handles PRs. It’s
+a custom DVC Git ref pointing to a Git commit. Here's an example.
+
+```dvc
+$ git show-ref exp-26220
+c42f48168830148b946f6a75d1bdbb25cda46f35 refs/exps/f1/37703af59ba1b80e77505a762335805d05d212/exp-26220
+```
+
+If you want to see your local experiments (that have not been pushed), you can
+run `dvc exp list --all`.
+
+You can read more about how we handle our custom Git refs in
+[this blog post](https://dvc.org/blog/experiment-refs).
+
+Thanks to @Chandana for asking this question about experiments!
+
+### [Q: Is there a way to list all the experiments I have on my DVC remote that have not been committed to Git?](https://discord.com/channels/485586884165107732/563406153334128681/836705209039978538)
+
+Yes! You can quickly look at all of the experiments with: `dvc exp list --all`.
+
+The command `dvc exp list --all origin` lists all of the experiments for a local
```

Comment:
This command will check the remote repo I think (that corresponds to the origin in the Git config)

Q @iterative/dvc was there a discussion to make `dvc exp list` list origin by default (otherwise `dvc exp list` default behavior is similar to the `dvc exp show`. It feels it makes sense to make this command more about remote? Just curious about thoughts on this. May be it's right to keep it explicit - similar to Git.

Reply:
> otherwise `dvc exp list` default behavior is similar to the `dvc exp show`

`dvc exp list` provides concise output that can be piped to other commands (see https://github.com/iterative/dvc/issues/5676 and https://github.com/iterative/dvc/issues/5489), so I think there at least needs to be an option for local.

Reply:
So is `dvc exp list --all` not the option for local?

Reply:
`dvc exp list --all` is the option for local, and `dvc exp list --all origin` or `dvc exp list --all <git repo URL>` are the options for the remote repo since `origin` is an alias for a git repo url.

You may not need to mention `dvc exp list --all` here since the question is about remotes.

Also, this sentence might be confusing since it is unclear if it is about the local or remote repo:

> The command `dvc exp list --all origin` lists all of the experiments for a local clone of the repo.

You might be able to condense the whole answer down to something like:

> Yes! You can quickly look at all of the experiments in any repo
> with `dvc exp list --all <git repo URL>` or `dvc exp list --all <git remote>`.

Reply:
That makes sense. Thanks for explaining that difference! Made that update.